### ISSUE TITLE
Allow choosing the transforms to apply

### ISSUE THREAD

**Issue by nene** [ga2mer/atom-lebab#2]
With upgrade to Lebab 2.0 (pull request #1) it no more makes sense to enable all the transforms by default.

The main culprit is the `exponent` transform for ES7, but one should in general be allowed to pick which transforms he wants to apply (like the `commonjs` won't be applicable if you want to limit yourself to what browsers natively support, or the `template` transform might too aggressively refactor your string concatenations).

I see two main ways of achieving this:
- Configuration option to specify the transforms to apply.
- Separate menu items for applying each individual transform.

**Comment by nene** [comment]
I saw that there is an alternative Lebab package for Atom: [atom-lebab-convert](https://github.com/jaunkst/atom-lebab-convert).

Unlike this package, it implements some configuration options, but I think it's inferior in the way it actually integrates with Lebab. So I proposed merging it into this project: https://github.com/jaunkst/atom-lebab-convert/issues/3

There probably isn't any actual code to bring over, but one could transfer some knowledge, like adopt a similar way of setting up configuration.
